### ISSUE TITLE
Symfony 4.x not supported. 

### ISSUE THREAD

**Issue by drekinov** [bugsnag/bugsnag-symfony#75]
### Expected behavior
Unhandled console exception to be registered by bugsnag.

### Observed behavior
Unhandled exceptions in console are not catched by bugsnag and developer end confused.

### Steps to reproduce
Install package and throw \Exception as first line in any console command

### Version
the one which claim to have symfony 4.x support

### Additional information
frustrated that i trusted that package marked to support symfony 4 will support it.
Especially when that package is "official for symfony by bugsnag" and it is about catching unhandled errors. If error is handled then who cares.
It was cool to debug whole app to find out that error is not registed in bugsnag not because my mistake but because package is not supporting symfony 4 at all.

Deprecation is noted in https://github.com/symfony/symfony/blob/4.2/UPGRADE-4.0.md 
"The console.exception event and the related ConsoleExceptionEvent class have been removed in favor of the console.error event and the ConsoleErrorEvent class."

**Comment by janmyszkier** [comment]
@drekinov please check my PR at https://github.com/bugsnag/bugsnag-symfony/pull/74
and let me know if that helps.
However, I just created a project with:
```
composer create-project symfony/skeleton blog
cd blog
composer require bugsnag/bugnag-symfony
```
and have no issues when running console commands. Can you elaborate on what exactly do you get?

**Comment by abigail-bugsnag** [comment]
A fix for this was released in [v1.5.1](https://github.com/bugsnag/bugsnag-symfony/releases/tag/v1.5.1).
